astronomer/providers/apache/livy/operators/livy.py

```diff
@@ -0,0 +1,98 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""This module contains the Apache Livy operator async."""
+from typing import Any, Dict
+
+from airflow.exceptions import AirflowException
+from airflow.providers.apache.livy.operators.livy import LivyOperator
+from airflow.utils.context import Context
+
+from astronomer.providers.apache.livy.triggers.livy import LivyTrigger
+
+
+class LivyOperatorAsync(LivyOperator):
+    """
+    This operator wraps the Apache Livy batch REST API, allowing to submit a Spark
```

Comment:
I think it would be helpful if we can add one point in docstring something like `Async (deferring) version of LivyOperator`

Reply:
@pankajastro the comment contains asynchronous implementation details:
` This operator wraps the Apache Livy batch REST API, allowing to submit a Spark
    application to the underlying cluster asynchronously.`

Reply:
make sense. I was just thinking to mention corresponding sync operator in docstring if it exists and the actual external API call we do in hook implementation so keeping API related message in the hook. But this is also ok.